src/clients/python/schemas/WebViewMessage.py

```diff
@@ -0,0 +1,68 @@
+# DO NOT EDIT: This file is auto-generated by generate-schema/index.ts
+from typing import Union
+import msgspec
+
+class StartedNotification(msgspec.Struct, tag_field="$type", tag="started"): 
+    version: str
+    """The version of the webview binary""" 
+
+class IpcNotification(msgspec.Struct, tag_field="$type", tag="ipc"): 
+    message: str
+    """The message sent from the webview UI to the client.""" 
+
+class ClosedNotification(msgspec.Struct, tag_field="$type", tag="closed"): 
+    pass 
+
+Notification  = Union[StartedNotification, IpcNotification, ClosedNotification] 
+""" 
+Messages that are sent unbidden from the webview to the client. 
+""" 
+class StringResultType(msgspec.Struct, tag_field="$type", tag="string"): 
+    value: str
+
+class BooleanResultType(msgspec.Struct, tag_field="$type", tag="boolean"): 
+    value: bool
+
+class FloatResultType(msgspec.Struct, tag_field="$type", tag="float"): 
+    value: float
+
+class undefined(msgspec.Struct, kw_only=True, omit_defaults=True): 
```

Comment:
suspicious `undefined` here

Reply:
Hmm, yeah, this is an issue with the discriminated union generation. In the rust code I'm using `content` to generate objects of type `{ $type: string, value: T}` and the `Size` arm of the enum isn't playing nice... Hmmmm.

https://github.com/zephraph/webview/blob/5b4d3263aa74c2a905ba151eb0169c7abd47b709/src/lib.rs#L272

Reply:
Got the base issue solved in #116 and fixed up the generation in this PR